### ISSUE TITLE
Map (M) and Minimap Struggling to load / Randomly Unloading Areas

### ISSUE THREAD

**Issue by ifozz** [anegostudios/VintageStory-Issues#4731]
### Game Version

v1.20.0 rc-5

### Platform

Windows

### Modded

Vanilla

### SP/MP

Singleplayer

### Description

Exactly what the title says. The map (M) is being really strange and difficult to work with.
You zoom out and pan around and some areas refuse to load, some load but then unload randomly, etc.

### Logs

```Log```

**Comment by Nephelangelo** [comment]
Can confirm. The map has been very sluggish to load which has been causing some frustration.

**Comment by ThePurpleDingo** [comment]
Agreed, the priority is too low. This bug could be titled "Major game feature gone" and still be accurate.
It ruins a part of the new content ; the map is integrated into the lore content, it is an issue that any player that decides to access the new locations is doomed to encounter.

**Comment by Thor8472** [comment]
I am very Close to rage quit! Sailing with this Bug ist nearly unplayable! Highest priority!

**Comment by Nephelangelo** [comment]
I want to add that this should probably be higher priority than “medium”. It’s been a problem for months now and renders a core part of the game sometimes unusable.

**Comment by zenmetsu** [comment]
I reverted to 1.19.8 and the problem was completely eliminated.  Highly recommend if you can deal with the missing features added in 1.20.

**Comment by LynnSnow13** [comment]
+1, a huge issue for us on our 1.20.6 server. What's worse is when the map fails to load, it's simultaneous with other huge drops in performance. Walking/running becomes jittery, and it affects other players who don't even have their maps open at the time.

**Comment by k1ngofpentacles** [comment]
Yeah plus one for this being a major impact on gameplay

**Comment by radfast** [comment]
Thanks to all who have reported and commented. I need greater clarity on what the issue is here. At present I am not able to reproduce issues of the types described, and if I can't reproduce them, I can't fix them.

1. Is it only the Map or Minimap which is unresponsive, or is the whole game being slow to actually load world chunks around the player?

2. Are the slow-loading areas previously generated and visited, or is this the first time visiting - i.e. is it the "bleeding edge" of the map that the player is moving towards, which is slow to load?

3. Is it a standard Survival world, single player, no mods?  Default world height and world sizes and landforms settings?  If not, please be specific about settings

4. Do I need to do something special with the map, like zoom out, pan it, or add markers, to see an issue?

**Comment by LynnSnow13** [comment]
1. The issue for me is triggered when viewing the map zoomed out and clicking and dragging it around faster than it can load in. If I stay zoomed in and move tiny bits at a time, it takes much longer to trigger. However, once triggered, the map "unloads", which also affects the minimap. When this happens, loading chunks stalls for a considerable amount of time (in my case, up to 10 minutes on our server).
2. It's both previously generated and bleeding edge for me.
3. I can test on single-player, but I currently play exclusively on a third-party hosted server with mods. I will edit this post when I get the worldgen settings and mod list to provide them.
4. Keep the map on the new visual style (storybookish instead of true color), zoom out, pan around relatively quickly on a large map. I have a ton of waypoints from some mod debug commands (4k plus), but everyone on our server has this issue and do not carry that number of waypoints and have been affected by this since we started 1.20 (very few waypoints). Interestingly, a few of us have switched back to the true color map style and have seen smoother loading and less of this unloading map/chunk unload issue.

**Comment by rarepops** [comment]
The thread in Discord has a lot of info https://discord.com/channels/302152934249070593/1327608582765674534

From my side, I am playing Multiplayer, on the TOPS server. I live at Z: -100k, maybe it is relevant but probably isn't.

The issue can be seen easily in my post, it reproduces 100% of the time for me if I scroll around while trying to use the Translocator
 https://discord.com/channels/302152934249070593/1327608582765674534/1356166152732020891

Another interesting piece of info is that the ping shoots up to 999+ corroborating the fact that events from your side, e.g. messages, attacking creatures, opening chests and similar things aren't being sent to the server; but they are being received - you can see messages from others, you can pick up items that are on the ground already. See https://discord.com/channels/302152934249070593/1327608582765674534/1363866175628513381
